custom_components/project_zero_three/sensor.py

```diff
@@ -55,8 +59,10 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     update_frequency = config[CONF_UPDATE_FREQUENCY]
     fuel_types = config[CONF_FUEL_TYPES]
+    region = config.get(CONF_REGION) #added region
 
-    data = FuelPriceData()
+    # data = FuelPriceData()
```

Comment:
can you remove commented out code please?

Reply:
My bad.. didn't know the change was on the master branch. I have updated the file to the original one.